### PR TITLE
Create keptn.yaml

### PR DESCRIPTION
**Pull request by maurosala**
## Hey, I just made a Pull Request!

Add Keptn plugin to the backstage plugin marketplace

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
FTR we are working on moving the plugin to the Keptn GitHub organization so that it becomes an official part of the project. FTR https://github.com/keptn-sandbox/contributing/issues/8 . Currently we need to confirm the license, AFAICT this is the only blocker for transfer

File: microsite/data/plugins/keptn.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: Keptn Plugin
+author: cloud@krateoplatformops.io
+authorUrl: https://github.com/krateoplatformops/krateo
+category: Monitoring
+description: Keptn plugin is designed for to monitoring a project and a single service on your keptn installation. You can also post a new delivery with the sync button!
+documentation: https://github.com/krateoplatformops/krateo/blob/main/README.md
+iconUrl: https://raw.githubusercontent.com/krateoplatformops/community/main/logos/white/logo_vertical.svg
+npmPackageName: '@krateo/backstage-plugin-keptn'
```
Comment (oleg-nenashev) [inline microsite/data/plugins/keptn.yaml:9]: Or `keptn/backstage-plugin`?

```suggestion
npmPackageName: '@keptn/backstage-plugin-keptn'
```

File: microsite/data/plugins/keptn.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+title: Keptn Plugin
+author: cloud@krateoplatformops.io
+authorUrl: https://github.com/krateoplatformops/krateo
+category: Monitoring
+description: Keptn plugin is designed for to monitoring a project and a single service on your keptn installation. You can also post a new delivery with the sync button!
+documentation: https://github.com/krateoplatformops/krateo/blob/main/README.md
+iconUrl: https://raw.githubusercontent.com/krateoplatformops/community/main/logos/white/logo_vertical.svg
```
Comment (oleg-nenashev) [inline microsite/data/plugins/keptn.yaml:8]:
```suggestion
iconUrl: https://raw.githubusercontent.com/keptn/community/master/logos/keptn-logo-square.svg
```

File: microsite/data/plugins/keptn.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+title: Keptn Plugin
+author: cloud@krateoplatformops.io
+authorUrl: https://github.com/krateoplatformops/krateo
+category: Monitoring
+description: Keptn plugin is designed for to monitoring a project and a single service on your keptn installation. You can also post a new delivery with the sync button!
```
Comment (oleg-nenashev) [inline microsite/data/plugins/keptn.yaml:6]:
```suggestion
description: Keptn plugin is designed for to monitoring a project and a single service on your Keptn installation. You can also post a new delivery with the sync button!
```

**Review by oleg-nenashev — CHANGES_REQUESTED** [review]
Suggesting changes based on moving the plugin to Keptn

**Comment by benjdlambert** [comment]
Seems we are missing a DCO check here too! Some help over in [CONTRIBUTING.md](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin) if you're unsure.

**Pull request closed without merging**